### PR TITLE
chore: Upload tests results to datadog WPB-2283

### PR DESCRIPTION
**Pull request by SzymonMrozek**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-2283" title="WPB-2283" target="_blank">TaskWPB-2283</a>  Create a metric for iOS tests flakiness
  </td></table>
  <br />
 

----

#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----
# What's new in this PR?

### Issues

We have a lot of flaky tests but don't have any metric of what are the most flaky tests

### Solutions

Use built-in metrics from DataDog
Screenshot 2023-06-27 at 13 28 45
Screenshot 2023-06-27 at 13 29 21

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios-data-model/.swiftlint.yml [hunk 1]
```diff
@@ -36,3 +36,5 @@ included:
 line_length:
     warning: 400
     error: 400
+
+
```
Comment (netbe) [inline wire-ios-data-model/.swiftlint.yml]: remove ?

File: wire-ios/Wire-iOS/Resources/Configuration/Wire-iOS/Wire-iOS-Base.xcconfig [hunk 2]
```diff
@@ -30,3 +30,4 @@ PROVISIONING_PROFILE_SPECIFIER = $(PROVISIONING_PROFILE_SPECIFIER_APP)
 
 // Apple Clang - Language
 GCC_PREFIX_HEADER = Wire-iOS/Wire-iOS-Prefix.pch
+
```
Comment (netbe) [inline wire-ios/Wire-iOS/Resources/Configuration/Wire-iOS/Wire-iOS-Base.xcconfig]: needed?

**Review by netbe — COMMENTED** [review]
just remove useless space / edited files

**Review by netbe — APPROVED** [review]

File: wire-ios-data-model/.swiftlint.yml (hunk 1 above)
Comment (SzymonMrozek) [reply wire-ios-data-model/.swiftlint.yml]: this is dummy to trigger tests 😄

File: wire-ios/Wire-iOS/Resources/Configuration/Wire-iOS/Wire-iOS-Base.xcconfig (hunk 2 above)
Comment (SzymonMrozek) [reply wire-ios/Wire-iOS/Resources/Configuration/Wire-iOS/Wire-iOS-Base.xcconfig]: this is dummy to trigger tests 😄

Comment (netbe) [reply wire-ios/Wire-iOS/Resources/Configuration/Wire-iOS/Wire-iOS-Base.xcconfig]: ok then merge it

**Review by agisilaos — APPROVED** [review]

**Pull request merged**